package.json

```diff
@@ -38,25 +38,25 @@
     "rollup.prod.ci": "rollup --config --config-prod --config-ci",
     "rollup.watch": "rollup --watch --config",
     "start": "npm run watch",
-    "test": "jest --coverage",
+    "test": "node --experimental-vm-modules ./node_modules/.bin/jest --coverage",
     "test.analysis": "cd test && npm run analysis.build-and-analyze",
     "test.bundlers": "cd test && npm run bundlers",
     "test.dist": "node scripts/build --validate-build",
     "test.end-to-end": "cd test/end-to-end && npm ci && npm test && npm run test.dist",
-    "test.jest": "jest",
+    "test.jest": "node --experimental-vm-modules ./node_modules/.bin/jest",
     "test.karma": "cd test/karma && npm ci && npm run karma",
     "test.karma.prod": "cd test/karma && npm ci && npm run karma.prod",
     "test.prod": "npm run test.dist && npm run test.end-to-end && npm run test.jest && npm run test.karma && npm run test.sys.node && npm run test.testing && npm run test.analysis",
     "test.testing": "node scripts/test/validate-testing.js",
-    "test.watch": "jest --watch",
-    "test.watch-all": "jest --watchAll --coverage",
+    "test.watch": "node --experimental-vm-modules ./node_modules/.bin/jest --watch",
+    "test.watch-all": "node --experimental-vm-modules ./node_modules/.bin/jest --watchAll --coverage",
```

Comment:
Are these `scripts` changes needed? Or are they bleed over from another branch?

Reply:
yep! prettier v3 uses ES modules internally (even if you use the cjs entry point) so when running prettier within Jest tests (which I added a little bit ago to make code output assertions more robust and easier) we get an error if we don't pass this

this is however failing on windows, so if you'd be able to try running it and see if you can clean anything the issue that would be super helpful!

Reply:
Bah, sorry, IDK how I missed https://github.com/ionic-team/stencil/pull/4563#discussion_r1258831079 originally 🤦